### ISSUE TITLE
Highlighted text after 140th char doesn't go normal after editing

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#155]
```
What steps will reproduce the problem?
1. Type more than 140 chars to get red highlighted text
2. Delete non highlighted text before 140th char

What is the expected output? What do you see instead?
Even if you go under the 140 threshold, the highlighted text won't get normal

What version of the product are you using? On what operating system?
Pino 0.2.4. Ubuntu 9.10

I attached a video showing the issue.
```

Original issue reported on code.google.com by `acunadit...@gmail.com` on 27 Mar 2010 at 6:28

Attachments:
- [out.ogv](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-155/comment-0/out.ogv)

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 27 Mar 2010 at 7:36
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by e92bfc0881 revision.
```

Original comment by `tro...@gmail.com` on 28 Mar 2010 at 2:35
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 169 has been merged into this issue.
```

Original comment by `tro...@gmail.com` on 31 Mar 2010 at 5:15
